### DISCUSSION TITLE
Vitocal 200-S mit Touch-Display (E08)

### DISCUSSION THREAD

**Discussion by ygoe** [openv/openv#653] [General]
Hallo,

mir wurde von der Baufirma die Wärmepumpe „Viessmann Vitocal 200-S AWB-M-E-AC 201.E08 NEV“ angeboten, und dazu Vitoconnect OPTO2. Letzteres lässt auf die Optolink-Schnittstelle schließen, aber ersteres scheint die gar nicht mehr zu haben. Stattdessen wird in der dazugehörigen Bedienungsanleitung beschrieben, wie man ohne jegliches Zubehör direkt eine WLAN-Verbindung zum Internet herstellen kann.

Das würde ja bedeuten, dass diese lokal „angreifbare“ Schnittstelle weggefallen ist und das System jetzt komplett verschlossen ist. Dann wäre ich auf jeden Fall vom Herstellerserver abhängig, was für mich inakzeptabel ist.

Habe ich was übersehen oder ist Optolink nun wirklich Vergangenheit? Also nicht, dass es jemals wirklich Gegenwart gewesen wäre. In früheren Versuchen mit anderen Geräten haben sich ja schon erhebliche Unzulänglichkeiten dieser optischen Schnittstelle und des Übertragungsprotokolls gezeigt. Ich habe hier auch den Sniffer gesehen, nachdem ich bereits selbst überlegt habe, ob sowas wohl möglich wäre; und da sind auch wieder Protokolländerungen beschrieben, die erst noch erforscht werden müssten. Da hätte ich mich eh nur ungern drauf eingelassen. Wenn es nun aber nicht mehr verfügbar ist, wird meine Entscheidung einfacher. Andere Hersteller bieten Modbus RTU/TCP oder lokale Web-APIs an. Ich weiß nicht, was Viessmann treibt, das wegzulassen.

**Comment by vich-667** [comment]
Hi!

Am Ende ist das alles hier wilde Bastelei, was zwar bei mir schon seit Jahren stabil läuft. Aber leider macht Viessmann am ende vom tag, das was alle anderen auch tun. WLAN rein und Verbindung in die Cloud, auf der sie zumindest eine API anbieten. Der Kundendiensttechniker greift auch per WLAN auf das gerät zu, fertig.

Ich hatte irgendwo mal nach einer LAN verbindung nachgefragt, da wurde mir dann ne Viessmann Vitogate 200 https://community.viessmann.de/viessmann/attachments/viessmann/customers-gas/10146/1/Vitogate_200_Typ_KNX_Montage-_und_Serviceanleitung.pdf empfohlen. Nicht ganz günstig der spaß aber eben noch local.

**Reply by ygoe** [reply]
Danke für den Hinweis. Das Vitogate 300 bietet wohl auch Modbus an, kostet aber so um die 1200 €. Oder man versucht, den internen(?) LON-Bus zu nutzen, über den das Vitogate angebunden ist. Dazu finden sich vereinzelt auch Beschreibungen, aber ein USB-LON-Adapter ist selten zu finden und kostet wohl auch so 350 €. Immerhin scheint es Optionen zu geben. Ob ich mich letztlich dafür entscheide, ist noch offen.
